### DISCUSSION TITLE
Welcome to Pony Origins Discussions!

### DISCUSSION THREAD

**Discussion by BlasterM5** [BlasterM5/ponyorigins#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lunar-Flare6234** [comment]
Hey, could you (or I) make a fork that uses powered elytra flight rather than creative flight? I'd like to use this alongside a mod that adds roll control in case somepony playing my pack wants to be a Wonderbolt

**Comment by BlasterM5** [comment]
Go for it. It's Unlicense so you can do whatever you want. Note I am about to push alpha 31 to maybe finally fix the eating issues.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Lunar-Flare6234** [reply]
That took way longer than it should have. Is there anywhere in particular you'd prefer I post it, or should I just send it to you so you reap the rewards since it's still mostly your work?

**Reply by BlasterM5** [reply]
Keep it on github so it can be referenced? I had originally considered making it work with elytra flight, but found it wasn't working with switching  between creative (for hover) and elytra (for sprinting), so since it was designed to be similar to how it works on BT server, I stuck with creative. That said, if elytra flight can be made to work smoothly like that, I'd be all over it.

**Reply by Lunar-Flare6234** [reply]
Pure elytra for now, I don't have the know how for that currently

**Reply by Lunar-Flare6234** [reply]
Fork is live. I also wanted to buff Rocket Mare since powered flight outpaces it but I couldn't figure out how

**Reply by Lunar-Flare6234** [reply]
So it turns out switching between elytra and creative flight is a vanilla feature (double tap space for creative>triple tap for elytra>double tap to go back to creative, or double tap with significant delay [well past the peak of your jump but before you land] to go straight to elytra) so you can just have both available by default; that also eliminates the need to make powered flight a thing, which gives rocket mare a purpose again without having to buff it.
